.github/workflows/publish.yml

```diff
@@ -0,0 +1,28 @@
+name: publish
+on: [workflow_dispatch]
+
+jobs:
+  build:
+    runs-on: [ubuntu-latest]
+    steps:
+      - name: checkout repository
+        uses: actions/checkout@v3
+      - name: validate gradle wrapper
+        uses: gradle/wrapper-validation-action@v1
+      - name: setup jdk 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: 17
+          distribution: 'microsoft'
+      - name: make gradle wrapper executable
+        run: chmod +x ./gradlew
+      - name: build
+        run: ./gradlew build
+      - name: publish
+        uses: Kir-Antipov/mc-publish@v3.3
+        with:
+          modrinth-id: RLrPqrNI
+          modrinth-token: ${{ secrets.MODRINTH_TOKEN }}
+          curseforge-id: 933674
+          curseforge-token: ${{ secrets.CURSEFORGE_TOKEN }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Missing trailing newline

Reply:
Updated in commit [f438132](https://github.com/orionzleon/EmuNO/commit/f438132c3aa8fb1c4dbdd96df5542b98207bbbe9), although a trailing newline is not necessary.